docs/docs/examples/guides/ur-harware-interface.md

```diff
@@ -0,0 +1,577 @@
+
+<div class="text--center">
+  <img src={urHWISequenceGraph} alt="Sequence application graph" />
+</div>
+
+Click on **Play** to run the AICA application. Repeat the same at the teaching pendant's screen to run the program. The
+robot moves through the positions and stops to hand over control to AICA Studio. The payload is adjusted, and control is
+handed back to the pendant.
+
+<div class="text--center">
+  <img src={urHWISequenceRunning} alt="Exchange of control" />
+</div>
+
+## Hand Guiding controller
+
+It is quite common that users need to manually adjust the position of the manipulator, either for practical -move to a
+part approach location and teach it to the robot- or safety reasons. While in Local mode, this can be achieved through
+the Freedrive function of the teach pendant, which is, however, in some senses limited. It demands users to press on the
+button behind the pendant and then adjust the configuration almost joint by joint, with the feedback being quite light
+and unresponsive at times.
+
+For that purpose, AICA offers a hand guiding controller as a part of the UR hardware collection, based on UR's native
+force mode, and enriched with additional functionality such as spatial limits. To use it, simply click on the **+** icon
+in the **Controllers** list, and select the **UR Hand Guiding Controller**.
+
+<div class="text--center">
+  <img src={urHWIHandGuidingGraph} alt="Hand guiding controller graph" />
+</div>
+
+Since the controller uses force mode, it needs the name and the reference frame of the force/torque (FT) sensor. In the
+settings of the controller set the name of the sensor as **ur_tcp_fts_sensor** and the frame as **ur_tool0**.
+
+:::note
+
+Attempts to guide the robot by pushing on individual links will fail, as the forces must act on the FT sensor on the end
+effector.
+
+:::
+
+After pressing **Play**, the manipulator can be hand guided to points in space, driven by the forces applied on its end
+effector. In other words, it reads forces in the FT sensor, and "admits" them, trying to set the measured force to zero.
+The controller can be further tuned and adjusted by using its parameters:
+
+- Velocity/Force limits: the velocities and forces that can be applied by the controller in force mode (X, Y, Z, RZ, RY,
+  RZ).
+- Force/Torque threshold: the thresholds above which the hand guiding behavior is activated.
+- Compliant axes selection: the axes along which the robot can be hand guided (1-enabled, 0-disabled).
+- Hold delay: forces below the thresholds above for this duration will disable hand guiding.
+- X/Y/Z limits: spatial boundaries for the end effector motion, vectors of two values for the lower and upper limit.
+- Reference orientation: quaternion representing a desired orientation in base frame
+- Angular limit: allowed deviation from the reference orientation
+- Linear/Angular boundary strength: gains to apply restitution forces/torques, in case linear or angular limits are
+  exceeded.
```

Comment:
I'd put parameters in bold to make them a bit more distinct and readable but that's just me. Up to your personal style